### DISCUSSION TITLE
Matek H743-Slim second gyro

### DISCUSSION THREAD

**Discussion by RipperGOT** [iNavFlight/inav#8131] [General]
Hello friends! Does anyone know why INAV with Matek H743-Slim V1.0 recognizes the second gyro (ICM20602) as MPU6500??? There is also no option to choose ICM20602 in the list of gyros. But if I choose the MPU6500 it works.

"Board V1.0 IMU: MPU6000 (SPI1) & ICM20602 (SPI4)"
"Board V1.5 IMU: MPU6000 (SPI1) & ICM42605 (SPI4)"

I have always configured the first gyro (MPU6000), but I am testing INAV 5.0 and wanted to try this gyro.

I asked in discord, but no one answered me. 😢

**Comment by DzikuVx** [comment]
https://github.com/iNavFlight/inav/blob/master/docs/Settings.md#gyro_to_use

**Comment by RipperGOT** [comment]
Thanks for your reply Pawel!!! My question is not how to configure it. I know how to do it.
My question is why the second gyro recognizes it as MPU6500. According to the Matek website the second gyro is an ICM20602, not an MPU6500
There is also no option to choose ICM20602 in the list of gyros


set looptime = 500
set gyro_main_lpf_hz = 70
set gyro_main_lpf_type = PT1
set dynamic_gyro_notch_q = 250
set dynamic_gyro_notch_min_hz = 90
**set gyro_to_use = 1**
set setpoint_kalman_q = 200
**set acc_hardware = MPU6500**
set acczero_x = -3
set acczero_y = -11
set acczero_z = 8
set accgain_x = 4086
set accgain_y = 4094
set accgain_z = 4083
set rangefinder_hardware = BENEWAKE
set rangefinder_median_filter = ON
set align_mag = CW90FLIP
set mag_hardware = QMC5883
